modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -160,14 +163,20 @@ public Response configureCamera(@HeaderParam("Accept") String acceptHeader, Came
 		Message message = new Message();
 
 		try {
+			Camera camera = cameraBeanWrapper.getOperation();
 
+			if (camera == null) {
+				throw new OperationManagementException("Camera bean is empty");
```

Comment:
Can we use more "context-friendly" message rather than saying "Camera *bean* is empty"?